sigar_linux_test.go

```diff
@@ -408,11 +408,15 @@ UdpLite: 0 0 0 0 0 0`
 			err := netStat.Get()
 			Expect(err).ToNot(HaveOccurred())
 			Expect(netStat.IP.InReceives).To(Equal(uint64(1055501)))
+			Expect(netStat.IP.InHdrErrors).To(Equal(uint64(33)))
 			Expect(netStat.IP.InAddrErrors).To(Equal(uint64(56)))
 			Expect(netStat.IP.ForwDatagrams).To(Equal(uint64(1)))
 			Expect(netStat.IP.InDelivers).To(Equal(uint64(1055445)))
 			Expect(netStat.IP.InDiscards).To(Equal(uint64(2)))
+			Expect(netStat.IP.InUnknownProtos).To(Equal(uint64(44)))
 			Expect(netStat.IP.OutRequests).To(Equal(uint64(503315)))
+			Expect(netStat.IP.OutDiscards).To(Equal(uint64(55)))
+			Expect(netStat.IP.OutNoRoutes).To(Equal(uint64(3)))
 
 			Expect(netStat.ICMP.InMsgs).To(Equal(uint64(83)))
 			Expect(netStat.ICMP.InErrors).To(Equal(uint64(1)))
```

Comment:
Further down, should you have a new assertion for the value of `netStat.TCP.CurrEstab`?